apps/condo/domains/organization/integrations/sbbol/SbbolFintechApi.js

```diff
@@ -260,7 +260,18 @@ const initSbbolFintechApi = async (userId) => {
     return fintechApi
 }
 
+const initSbbolClientWithToken = (accessToken) => {
```

Comment:
No. because there is such a function above. It creates a client by userId, and this function creates a client by accessToken. The name reflects what the function is for.